iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/MultiplexingClientTests.java

```diff
@@ -1166,20 +1176,27 @@ public void multiplexedConnectionRecoversFromDeviceSessionDropsParallel() throws
     }
 
     // Open a multiplexed connection, send a fault injection message to drop the TCP connection, and ensure that the multiplexed
-    // connection recovers
+    // connection recovers and that no twin/method subscriptions were lost
```

Comment:
Adding this additional check to the test since we didn't have coverage for this scenario already